contract/tracking.proto

```diff
@@ -62,6 +62,7 @@ enum Subject {
   SUBJECT_MOBILE_SIM = 7;
   SUBJECT_SMART_METER_INSTALLATION = 8;
   SUBJECT_CUSTOMER_OVERDUE_BALANCE = 9;
+  SUBJECT_INSURANCE_QUOTING = 10;
```

Comment:
would this be very specific like "home insurance quoting"
Or can I leave it slightly generic as it could be used for Boiler too.

In the future there could be even more types of policies to quote for so... I dont know what is best practice here